src/uri.cr

```diff
@@ -59,10 +59,10 @@ class URI
   # ```
   # URI.parse("http://foo.com/bar").path # => "/bar"
   # ```
-  getter path : String?
+  getter path : String
```

Comment:
Why not use the `property` syntax?
It seems like the `getter` and `setter` are the same.

Reply:
Because of documentation (setter and getter need different docs).